tools/ci/alibaba-mirror-settings.xml

```diff
@@ -24,5 +24,11 @@ under the License.
       <url>http://172.17.0.1:8888/repository/maven-central/</url>
       <mirrorOf>central</mirrorOf>
     </mirror>
+    <mirror>
+      <id>alicloud-mvn-confluent-mirror</id>
+      <name>Alibaba Confluent Maven mirror</name>
+      <url>http://172.17.0.1:8888/repository/confluent/</url>
```

Comment:
Does Alibaba provide the https variant of the maven proxy?

Also do they have a DNS record available for the proxy by any chance?

Reply:
This is something that we have setup ourselves on the machines, so no to both.

Reply:
👍